lambdas/submit-answer/src/submit-answer-handler.ts

```diff
@@ -50,12 +52,17 @@ export class SubmitAnswerHandler implements LambdaInterface {
 
       const verificationScore: number =
         verificationScoreCalculator.calculateVerificationScore(answerResult);
+
+      const ttl = event.usersQuestions.Items[0].expiryDate as number;
+
       await this.resultService.saveResults(
         event.sessionId,
         event.dynamoResult.Item.correlationId.S,
-        event.usersQuestions.Items[0].expiryDate,
+        ttl,
         answerResult,
-        verificationScore
+        verificationScore,
+        checkDetailsCountCalculator.calculateCorrectAnswers(answerResult),
```

Comment:
Can we pull these to variables so if anything fails its easier to identify which line

Reply:
Like we have for ttl